cluster/kubemark/gce/config-default.sh

```diff
@@ -36,8 +36,8 @@ HOLLOW_PROXY_TEST_LOG_LEVEL="${HOLLOW_PROXY_TEST_LOG_LEVEL:-$TEST_CLUSTER_LOG_LE
 TEST_CLUSTER_API_CONTENT_TYPE="${TEST_CLUSTER_API_CONTENT_TYPE:-}"
 
 # Hollow-node components' test arguments.
-HOLLOW_KUBELET_TEST_ARGS="${HOLLOW_KUBELET_TEST_ARGS:-} ${HOLLOW_KUBELET_TEST_LOG_LEVEL}"
-HOLLOW_PROXY_TEST_ARGS="${HOLLOW_PROXY_TEST_ARGS:-} ${HOLLOW_PROXY_TEST_LOG_LEVEL}"
+HOLLOW_KUBELET_TEST_ARGS="${HOLLOW_KUBELET_TEST_ARGS:-${HOLLOW_KUBELET_TEST_LOG_LEVEL}}"
```

Comment:
Those changes are incorrect - we want to set log-level even if args are set.

Reply:
I found that this script will run 2 times, and the value of `HOLLOW_KUBELET_TEST_ARGS` will become "--v=4 --v=4", this won't work well in the pod command spec, the command args should be one by one.

Reply:
Now I use a different way in the hollow node command, we don't need to change the code here now.